client/gutenboarding/gutenboard.tsx

```diff
@@ -40,30 +38,46 @@ export function Gutenboard() {
 		updateIsEditorSidebarOpened( ! isEditorSidebarOpened );
 	}
 
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
-		<div className="gutenboarding">
-			<Header
-				isEditorSidebarOpened={ isEditorSidebarOpened }
-				toggleGeneralSidebar={ toggleGeneralSidebar }
-			/>
+		<div className="gutenboarding block-editor__container">
 			<SlotFillProvider>
 				<DropZoneProvider>
-					<BlockEditorProvider value={ [ onboardingBlock ] } settings={ { templateLock: 'all' } }>
-						<div className="gutenboarding__block-editor">
-							<BlockEditorKeyboardShortcuts />
-
-							<SettingsSidebar isActive={ isEditorSidebarOpened } />
-							<Sidebar.Slot />
-							<WritingFlow>
-								<ObserveTyping>
-									<BlockList />
-								</ObserveTyping>
-							</WritingFlow>
-						</div>
-						<Popover.Slot />
-					</BlockEditorProvider>
+					<div
+						className={ classnames( 'edit-post-layout', {
+							'is-sidebar-opened': isEditorSidebarOpened,
+						} ) }
```

Comment:
>These are classes that Gutenberg uses in the editor.

Yep! It's basically a choice between the two approaches: import CSS and re-use classes or copy-paste a bunch of CSS from the core.

I think both approaches are valid as a mid-step but since neither is ideal (using core components to produce a layout would be ideal instead), I think re-using classes is something that's little easy to keep up with in core. I also tried limiting number of classes we re-use.